core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -16,6 +16,8 @@
  */
 package org.apache.calcite.test;
 
+import jdk.nashorn.internal.ir.annotations.Ignore;
```

Comment:
Hmm... First time to see Ignore from nashorn, not from JUnit.

Reply:
Just noticed, Intellij decided to import something weird. Will fix soon.
Thanks for pointing this out!